flags.ts

```diff
@@ -17,7 +17,15 @@ export const cookies = {
     const cookies = getCookies(headers);
     const flags: CookiedFlag[] | undefined = Object.keys(cookies)
       .filter((cookie) => cookie.startsWith(DECO_COOKIE))
-      .map((cookie) => JSON.parse(atob(cookies[cookie])));
+      .map((cookie) => {
+        try {
+          return JSON.parse(atob(cookies[cookie]));
+        } catch (err) {
+          console.error("error parsing cookies", err);
+          return undefined;
```

Comment:
Not sure, I'm afraid that because of a bug on how we parse the cookie we could erase cookies that are actually valid but our code is buggy. WDYT?

Reply:
I was concern about a broken live version that inserts invalid cookies will made the parser always fail. Clearing it would allow those users to get a new (correct) cookie.

I can now see that this may be fixable by adding a versioning at the `DECO_COOKIE` prefix instead.

So, we should keep this code as is for now.